src/backend/db/products.js

```diff
@@ -1,30 +1,36 @@
-import { v4 as uuid } from "uuid";
+import { v4 as uuid } from 'uuid';
 
 /**
  * Product Database can be added here.
  * You can add products of your wish with different attributes
  * */
-
+import { realMadrid, football, manchesterUnited } from './../../assets/index';
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    title: 'Real Madrid Jersey',
+    price: '5000',
+    categoryName: 'jersey',
+    inWishlist: true,
+    imgSrc: realMadrid,
+    isBestSeller: true,
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    title: 'Nike Strike Football',
+    price: '3000',
+    inWishlist: false,
+    categoryName: 'football',
+    inWishlist: false,
```

Comment:
You have repeated inWishlist two times over here.

Reply:
thanks for the feedback atul.
will correct it.👍